### PR TITLE
chore(documentation): remove documentation v6 from the repo

### PR DESCRIPTION
**Pull request by gfellerph**
Removes the v6 storybook documentation package and it's associated workflows and scripts.

eldorado-begone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,13 +25,6 @@
     "docs:test": "pnpm --filter design-system-documentation test",
     "docs:e2e": "start-server-and-test docs:headless 9300 'pnpm --filter design-system-documentation e2e'",
     "docs:e2e:watch": "start-server-and-test docs:headless 9300 'pnpm --filter design-system-documentation e2e:watch'",
-    "docsv6": "pnpm --filter design-system-documentation-v6 start",
-    "docsv6:start": "pnpm --filter design-system-documentation-v6... --parallel --stream start",
-    "docsv6:build": "pnpm --filter design-system-documentation-v6 build",
-    "docsv6:headless": "pnpm --filter design-system-documentation-v6 start:headless",
-    "docsv6:e2e": "start-server-and-test docsv6:headless 9300 'pnpm --filter design-system-documentation-v6 e2e'",
-    "docsv6:e2e:watch": "start-server-and-test docsv6:headless 9300 'pnpm --filter design-system-documentation-v6 e2e:watch'",
-    "docsv6:snapshots": "start-server-and-test 'pnpm docsv6:headless' 9300 'pnpm --filter design-system-documentation-v6 snapshots'",
```
Comment (oliverschuerch) [inline package.json:34]: Don't we need the snapshot script for the current docs package?

**Review by oliverschuerch — APPROVED** [review]

**Comment by oliverschuerch** [comment]
Do not forget to remove the netlify v7 url afterwards!

File: package.json (hunk 1 above)
Comment (gfellerph) [reply package.json:34]: There's still the "snapshots" command that directly invokes the script in the documentation/package.json.

**Comment by gfellerph** [comment]
> Do not forget to remove the netlify v7 url afterwards! https://app.netlify.com/sites/swisspost-design-system-next-v7/configuration/general#danger-zone

I'd actually keep it online just for preserving the URL. If we delete the page, someone else can come in and use that slug.

**Pull request merged**